mux/src/pane.rs

```diff
@@ -118,6 +118,7 @@ pub trait Pane: Downcast {
     fn palette(&self) -> ColorPalette;
     fn domain_id(&self) -> DomainId;
 
+    #[allow(unused)]
     fn erase_scrollback(&self, erase_mode: ScrollbackEraseMode) {}
```

Comment:
I hope this alright. I figured it's better to allow `erase_mode` to be unused than prefix it with `_`

Reply:
I'd prefer `_erase_mode` here; attributes are more to type and can diverge from the code that led to the attribute being added.

The `_` prefix is understood to mean that it is unused in a given instance, even though it feels a little weird for a trait definition to have unused parameters.  This is a little bit of a wart in Rust TBH.

Reply:
No problem. Would you like / be okay with me to rebase as well? I can change `erase_mode` to `_erase_mode` in the initial commit I added it. I don't want you to lose your PR comments unless this is the last thing before you are ready to merge.

Reply:
I'm totally fine with you rebasing/squashing/force pushing.  My next steps here when I've got a few minutes in the right headspace later this evening are:

* Actually look closely at the screen/line buffer manipulation (I've only glossed over it so far)
* Assuming it's good (nothing obviously bad in my skimming so far), squash and merge
* Add an entry in the changelog; briefly summarizing the change; you're welcome to preempt me on that!